### DISCUSSION TITLE
Android notifications randomly not delivered

### DISCUSSION THREAD

**Discussion by renatomserra** [expo/expo#31427] [Archived: Questions]
Since the move from FCM legacy a large portion of my android messages are not being delivered, but both send and receipt say they are ok.
Anyone else experiencing this?

Example receipt ID `0191e3be-026d-7470-94bf-9dcd8802e240` , `0191e3c5-9f67-754f-9a54-c7ab5957f967`, `"0191e4e7-5f58-784c-80f8-e1de53a4c942`

With 20k MAU without any app or server changes, just updating my credentials since the move, now its very flakey getting lots of complaints, Ive been able to reproduce with the above receipt IDs

My app sends medication and appointment reminders set by users, so they notice straight away.

**Comment by namledev056** [comment]
The same issue. what happen with android

**Comment by aminosman** [comment]
Same issue, status returned is OK but from test a random ~10% are not actually received by the user
